warehouse/templates/packaging/detail.html

```diff
@@ -210,7 +210,10 @@ <h2 class="page-title split-layout">
               </div>
 
               <a class="card release__card" href="{{ request.route_path('packaging.release', name=release.project.name, version=hversion.version) }}">
-                <p class="release__version">{{ hversion.version }}</p>
+                <p class="release__version">
+                  {{ hversion.version }}
+                  {% if hversion.is_prerelease %}<span class="badge badge--warning">pre-release</span>{% endif %}
```

Comment:
To follow BEM convention, we should add a class here called `release__version-badge` and add it into the SCSS.